README.md

```diff
@@ -404,6 +404,71 @@ sudo hab svc load core/artifactory
 ```
 This will spin up an Artifactory instance, and you can use the following to log in: http://localhost:8081/artifactory/webapp/#/home
 
+### Backing up Builder Data
+
+The data that Builder stores is luckily fairly lightweight and thus the backup and DR strategy is pretty straightforward. With On-Prem Builder we have two types of data that a user may want to restore in the case of a disaster. First, the package and user metadata, which gets stored in a PostgreSQL instance and second, habitat artifacts (.harts) which get stored in whichever artifact storage backend you've chosen to use (Minio, S3, Artifactory).
+
+Ideally backups of the Builder cluster would be performed in lock-step however, due to the nature of the data that Builder stores there is very little concern with the timing of this operation. In the worst case if a package's metadata is missing from PostgreSQL, it can easily be repopulated by re-uploading the package with the `--force` flag like so `hab pkg upload <path to hartfile> -u <on-prem_url> --force`
+
+#### Database Backups
+
+The process of backing up Builder's PostgreSQL database should be no different than backing up any other PostgreSQL database. The process amounts to a pg_dump. If you have a backup strategy for other production instances of PostgreSQL we suggest you use the same strategy here with the `builder` database. Otherwise the process of a manual backup would be as follows:
+
+1. Shut down the API to ensure no active transactions are occuring. (Optional but preferred)
+        `hab svc stop habitat/builder-api`
+1. Switch to user `hab`
+        `sudo su - hab`
+1. Find your Postgres password 
+        `sudo cat /hab/svc/builder-api/config/config.toml`
+1. Export as envvar 
+        `export PGPASSWORD=<pw>`
+1. Run pgdump 
+        `/hab/pkgs/core/postgresql/<version>/<release>/bin/pg_dump --file=builder.dump --format=custom --host=<ip_of_pg_host> --dbname=builder`
+1. Start the api and verify 
+        `sudo hab svc start habitat/builder-api`
+
+Now the backup of your database has been dumped into the `builder.dump` file on your filesystem and can be moved or stored in whatever manner you see fit. For the backup to be useful in a DR scenario we obviously suggest moving the database dump somewhere else. An AWS S3 bucket would suffice for many but again we suggest you store the dump with the same strategy in which you store any other database dumps.
```

Comment:
```suggestion
Once the backup finishes,  your archive file will be named `builder.dump` file on your filesystem. Move and store this file according to your local policies. We recommend storing it remotely, either physically or virtually, so it will be useable in a worst-case scenario. For most, storing the dump file in an AWS bucket or Azure storage is enough, but you should follow the same strategy outlined in your disaster recovery plan for all database dumps. 
```